src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -48,11 +48,12 @@ public Instant getTimestamp() {
 
     @SuppressWarnings("unused, used from DAO")
     @JsonProperty("item-hash")
+    @JsonIgnore
```

Comment:
That's not exactly what I'm asking, I should have been more explicit. By marking this property with `@JsonIgnore` we're effectively removing it from being included in the download formats, which is the breaking change. Do we want to do this or have a period of time where an entry consists of both `item-hash` and `item-hashes`?